data/data/bootstrap/files/usr/local/bin/installer-masters-gather.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+ARTIFACTS="${1:-/tmp/artifacts}"
+mkdir -p "${ARTIFACTS}"
+
+echo "Gathering master journals ..."
+mkdir -p "${ARTIFACTS}/journals"
+for service in bootkube openshift kubelet crio
+do
+    journalctl --boot --no-pager --output=short --unit="${service}" > "${ARTIFACTS}/journals/${service}.log"
+done
+
+echo "Gathering master containers ..."
+mkdir -p "${ARTIFACTS}/containers"
+for container in $(crictl ps --all --quiet)
```

Comment:
If ShellCheck is happy, I guess I'm happy, but I'd have expected you'd need to pipe into `while read -r ...` here like we do above.